src/download.ts

```diff
@@ -15,140 +12,99 @@ import * as path from 'path';
 import { SimpleExecutor } from './executor';
 import { logOutputChannel, outputChannel } from './log';
 
+export class DownloadError extends Error {
+  constructor(message: string, readonly url: URL, readonly expectedCliPath: string) {
+    super(message);
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const EXPECTED_DEVTOOLS_VERSION = 'c0b212c6be46a95802666aaf0a18678b055feb1e';
+
 const artifacts: { [platform: string]: { [arch: string]: string } } = {
   darwin: {
-    x64: 'codescene-cli-ide-macos-amd64-v4.zip',
-    arm64: 'codescene-cli-ide-macos-aarch64-v4.zip',
+    x64: `codescene-cli-ide-macos-amd64-${EXPECTED_DEVTOOLS_VERSION}.zip`,
+    arm64: `codescene-cli-ide-macos-aarch64-${EXPECTED_DEVTOOLS_VERSION}.zip`,
```

Comment:
Maybe should be called required dectools version?

Reply:
Yeah thanks, I got stuck in naming limbo for a while there...